core/models.py

```diff
@@ -1,3 +1,31 @@
 from django.db import models
+from django.utils.datetime_safe import datetime
+from django.template.defaultfilters import slugify
+from django.utils.translation import ugettext as _
 
-# Create your models here.
+
+# Post model
+class Post(models.Model):
+    title = models.CharField(max_length=255, null=False, blank=False, verbose_name=_('Заголовок'))
+    body = models.TextField(null=False, blank=False, verbose_name=_('Тест поста'))
+    created_at = models.DateTimeField(null=False, blank=False, verbose_name=_('Создано в'))
```

Comment:
I think the [auto_now_add](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DateField.auto_now_add) excellently satisfying creation timestamps fields.